src/ontodia/data/rdf/rdfLoader.ts

```diff
@@ -0,0 +1,85 @@
+import { RDFGraph } from 'rdf-ext';
+
+import 'whatwg-fetch';
```

Comment:
I think we shouldn't put `import 'whatwg-fetch'` into each file and instead do it once at the `src/ontodia/index.ts`.